src/context/AuthContext.tsx

```diff
@@ -71,7 +71,7 @@ export const AuthProvider: ChildrenFC = ({ children }) => {
     setUser(emptyUser);
   }, []);
 
-  const isUserLoggedIn = user.id !== '-1';
+  const isUserLoggedIn = user.id !== '';
```

Comment:
@GLObus303 we have ` emptyUser` now... but if we will default to null, I will keep it for the next PR.